pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -1,146 +1,91 @@
 { lib
-, stdenv
-, python
-, cmake
-, meson
-, vim
-, ruby
-, which
+
+  # nixpkgs functions
+, buildGoModule
+, buildVimPluginFrom2Nix
 , fetchFromGitHub
-, fetchurl
 , fetchpatch
-, llvmPackages
-, rustPlatform
-, buildGoModule
-, pkg-config
-, curl
-, openssl
-, libgit2
-, libiconv
-, xkb-switch
-, fzf
-, skim
-, stylish-haskell
+, fetchurl
+, stdenv
+, substituteAll
+
+  # Language dependencies
+, python
 , python3
+, rustPlatform
+
+  # Misc dependencies
+, Cocoa # vim-addon-manager
+, code-minimap # minimap-vim
+, dasht # vim-dasht
+, direnv # direnv-vim
+, fzf # fzfWrapper
+, gnome3 # vCoolor-vim
+, khard # deoplete-khard
+, languagetool # lens-vim, vim-grammarous
+, llvmPackages # clang_complete, clighter8
+, meson # meson
+, nim # fruzzy
+, nodePackages # coc-*
+, skim # skim, skim-vim
+, sqlite # sql-nvim
+, stylish-haskell # vim-stylish-haskell
+, tabnine # compe-tabine, completion-tabnine
+, vim # unicode-vim, vim-hier
+, which # vimproc-vim
+, xkb-switch # vim-xkbswitch
+, ycmd # YouCompleteMe
```

Comment:
I wouldn't want to manage those as they are quickly out of date. We can use nixpkgs-hammering to find unused ones.

Reply:
Neat, I didn't know about `nixpkgs-hammering`. I'll remove to comments.